app/controllers/static_pages_controller.rb

```diff
@@ -1,6 +1,14 @@
 class StaticPagesController < ApplicationController
   def home
-    @pagy, @products = pagy Product.all.newest, items: Settings.pagy.item_9
+    if params
+      @products = Product
+      if params[:starts_with].present?
+        @products = @products.filter_by_starts_with(params[:starts_with])
+      end
+    else
+      @products = Product.all.newest
+    end
```

Comment:
```suggestion
@products = Product.products.filter_by_starts_with(params[:starts_with]).newest
```
k cần check param vì nếu param trống nó sẽ trả về all record